src/python/bot/tasks/fuzz_task.py

```diff
@@ -1241,6 +1242,35 @@ def key_fn(crash):
   return new_crash_count, known_crash_count, processed_groups
 
 
+# TODO: remove environment variable once refactor is complete
+# Set multi-armed bandit strategy selection distribution as an environment
+# variable so we can access it in launcher.
+def set_strategy_distribution_in_env():
+  """Read FuzzStrategyProbability ndb table and set data to environment
+  variable."""
+  if environment.get_value('USE_BANDIT_STRATEGY_SELECTION'):
+    query = data_types.FuzzStrategyProbability.query()
+    distribution = []
+    for strategy_entry in list(ndb_utils.get_all_from_query(query)):
+      distribution.append({
+          "strategy_name": strategy_entry.strategy_name,
+          "probability": strategy_entry.probability
+      })
+    environment.set_value('STRATEGY_SELECTION_DISTRIBUTION', distribution)
+
+
+def get_strategy_distribution_from_ndb():
```

Comment:
This function can be deleted, right? I think some tests might need to be updated but otherwise I don't see a good reason to have both this and the above.